lib/Github/Api/PullRequest.php

```diff
@@ -14,6 +14,31 @@
  */
 class PullRequest extends AbstractApi
 {
+    use AcceptHeaderTrait;
+
+    /**
+     * Configure the body type.
+     *
+     * @link https://developer.github.com/v3/pulls/#custom-media-types
+     * @param string|null $bodyType
+     */
+    public function configure($apiVersion = null, $bodyType = null)
```

Comment:
Correct, this was to allow for setting the preview version of https://developer.github.com/v3/pulls/#merge-a-pull-request-merge-button

Looking now, the order doesn't make much sense as I did it, I'm guessing that I probably started on the PullRequest/Comments (since that is what I was looking to set a preview version for and then came back through later when I figured I would update all of the places that allow changing the bodyType of message responses.  I'll update this to flip the order around (and also to add the missing `@param` that I forgot to include when copy/pasting the comment block)